channels/stable

```diff
@@ -58,11 +58,11 @@ spec:
       providerID: aws
       architectureID: amd64
       kubernetesVersion: ">=1.17.0 <1.18.0"
-    - name: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20211015
+    - name: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20211118
```

Comment:
I wonder if we should go to 10/21 first. But maybe that's not important for AMIs.

Reply:
Not very important. Already updated in test and don't see any issues.